cocotb/share/lib/vpi/VpiImpl.h

```diff
@@ -65,9 +65,8 @@ static inline int __check_vpi_error(const char *file, const char *func, long lin
             loglevel = GPIWarning;
     }
 
-    gpi_log("cocotb.gpi", loglevel, file, func, line,
-            "VPI Error %s\nPROD %s\nCODE %s\nFILE %s",
-            info.message, info.product, info.code, info.file);
+    gpi_log("cocotb.gpi", loglevel, file, func, line, "VPI error");
+    gpi_log("cocotb.gpi", loglevel, info.file, info.product, info.line, info.message);
```

Comment:
```suggestion
    gpi_log("cocotb.gpi.vpi", loglevel, info.file, info.product, info.line, info.message);
```

We don't put the interface in the logger name anywhere in the C code, but there is no reason we shouldn't.

Reply:
What does `info.product` contain here?

Reply:
Did you deliberately suggest removing the first log here @ktbarrett? Without it, our line number does not escape.

Reply:
info.product returns 'verilator', which makes it clear that the error comes from verilator. 
Looking at the example log above, I think we want to keep both log lines to give all the context.

Reply:
> Without it, our line number does not escape.

Ahh, good point. Nvm.